.github/workflows/main.yml

```diff
@@ -0,0 +1,51 @@
+name: Upload Python Package
+
+# on:
+#   release:
+#     types: [created]
+
+on:
+  push: 
+    branches:
+    - dciborow-patch-1
+    - release/*
+    
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+    - name: Build and publish
```

Comment:
i just found this action, should be able to us it here
https://github.com/marketplace/actions/pypi-publish